readme.md

```diff
@@ -25,6 +25,21 @@ isAbsoluteUrl('foo/bar');
 //=> false
 ```
 
+## Options
```

Comment:
Use the same format for readme docs as used here: https://github.com/sindresorhus/boxen#api